api/indexer.oas2.json

```diff
@@ -917,6 +957,16 @@
         "is-frozen": {
           "description": "\\[f\\] whether or not the holding is frozen.",
           "type": "boolean"
+        },
+        "opted-in-at-round": {
+          "description": "Round during which the account opted into this asset holding.",
+           "type": "integer",
+           "x-algorand-format": "uint64"
+        },
+        "opted-out-at-round": {
+          "description": "Round during which the account opted out of this asset holding.",
+           "type": "integer",
+           "x-algorand-format": "uint64"
```

Comment:
AssetHolding
* OptedInAtRound
* OptedOutAtRound - This should probably technically be "ClosedOutAtRound" because we close them with a closeTo transaction similar to an account.

These are the most inconsistent in our system.
Python, Javascript, goal - no special name, just special arguments to the AssetTransferTransaction
Go - `MakeAssetAcceptanceTxn`
Java - `AssetAcceptTransactionBuilder().acceptingAccount`